.docker/localstack/dynamodb/annotations_api_highlight_notes.json

```diff
@@ -12,6 +11,28 @@
     {
       "AttributeName": "highlightId",
       "AttributeType": "S"
+    },
+    {
```

Comment:
would it be better to define primary key as composite, i.e sort key as highLightId and partition key as userId.

we can then retrieve item by userId to delete the PII for that user, and skip the GSI for the userId

Reply:
yeah, the only bummer is that it might result in hot partitions since we could have notes power users who leave a lot of notes...